### PR TITLE
Refactor: Convert to new Solo Project format

### PR DESCRIPTION
**Pull request by jdmedlock**
Convert to new Solo Project format

Resolves: N/a
See also: N/a

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,23 +3,69 @@
...
+to build a web application using the Mapbox API to build a map of a city of 
+your choice.
 
 ## Instructions
 
-General instructions for all Pre-Work Projects are located in the Chingu Voyage Handbook (URL posted in the
-`#read-me-first` channel on Discord). For detailed requirements checkout the ZenHub Board that is part of this repo.
+General instructions for all Solo Projects are located in the 
+[Chingu Library](https://voyage.docs.chingu.io/prework/howwork). For detailed 
+requirements checkout the following requirements.
```
Comment (RavalVarun) [inline readme.md]: For detailed requirements, please checkout the following:

Comment (RavalVarun) [inline readme.md]: For detailed requirements, please checkout the following:

File: readme.md [hunk 2]
```diff
@@ -3,23 +3,69 @@
...
-To see the ZenHub tab on the GitHub repo webpage requires that you install ZenHub's browser extension. For more
-information visit the [ZenHub Browser Extension page](https://www.zenhub.com/extension) and these 
-[installation instructions](https://t.ly/3v8w).
+*Structure*
+- [ ] Read over the Mapbox API documentation
 
-If you would like to add these tasks to your repo so you can track the progress of your Solo Project and so they will be 
-more readily available for reference you will need to manually add new issues to your ZenHub board and copy/paste the text
-in our issues to yours.
+      Things to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:
```
Comment (RavalVarun) [inline readme.md]: Points to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:

Comment (RavalVarun) [reply readme.md]: or Topics to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:

File: readme.md [hunk 3]
```diff
@@ -3,23 +3,69 @@
...
+  - [ ] Map
+- Create a sidebar with the following elements:
+  - [ ] Search input field
+  - [ ] A list of search results (should match the markers on the map)
+  - [ ] A button to toggle the display/hide of the sidebar
+- Create a map component to contain the map of your city and location markers
+
+*Style*
+- [ ] You may use any style you choose. However, it should be consistent (e.g.
+font, font size, color scheme, layout, etc.).
```
Comment (RavalVarun) [inline readme.md:40]: Should we provide a link to any article which gives some details about consistent styling?

File: readme.md [hunk 4]
```diff
@@ -3,23 +3,69 @@
...
+
+*Functionality*
+- Sidebar behaviors:
+  - [ ] By default all all available markers should be displayed
```
Comment (RavalVarun) [inline readme.md]: - [ ] By default all available markers should be displayed

File: readme.md [hunk 5]
```diff
@@ -3,23 +3,69 @@
...
+  - [ ] Markers update in search results and on the map while the user is 
+  typing intp the search input field
```
Comment (RavalVarun) [inline readme.md]: looks like typo for "intp"

File: readme.md [hunk 6]
```diff
@@ -3,23 +3,69 @@
...
+  - [ ] User should be able to click a button to toggle the sidebar visibility
+- The map component displays:
```
Comment (RavalVarun) [inline readme.md]: The map component should display:

File: readme.md [hunk 7]
```diff
@@ -3,23 +3,69 @@
...
+  - [ ] The map of your city using the Mapbox API
+  - [ ] At least five markers for five unique locations or attractions in your city
+
+*Other*
+- [ ] Your repo needs to have a robust `README.md`
```
Comment (RavalVarun) [inline readme.md]: should we add the link to the article https://medium.com/chingu/keys-to-a-well-written-readme-55c53d34fe6d

File: readme.md [hunk 8]
```diff
@@ -3,23 +3,69 @@
...
+- [ ] Make sure that there are no errors in the developer console before submitting
```
Comment (RavalVarun) [inline readme.md]: Before submitting, make sure that there are no errors in the developer console

File: readme.md (hunk 1 above)
Comment (jdmedlock) [reply readme.md]: Update applied.

File: readme.md (hunk 2 above)
Comment (jdmedlock) [reply readme.md]: Update applied

File: readme.md (hunk 3 above)
Comment (jdmedlock) [reply readme.md:40]: Good idea! Adding the following:
  - [Consistent Web Design](https://1stwebdesigner.com/consistent-web-design/)
  - [Why is consistency important in Web Design?](https://laceytechsolutions.co.uk/blog/importance-of-consistency-in-web-design/)

File: readme.md (hunk 4 above)
Comment (jdmedlock) [reply readme.md]: Update applied

File: readme.md (hunk 5 above)
Comment (jdmedlock) [reply readme.md]: Update applied

File: readme.md (hunk 6 above)
Comment (jdmedlock) [reply readme.md]: Update applied

File: readme.md (hunk 7 above)
Comment (jdmedlock) [reply readme.md]: Update applied

File: readme.md (hunk 8 above)
Comment (jdmedlock) [reply readme.md]: Update applied

**Pull request merged**